src/main/java/dk/kb/storage/util/DsStorageClient.java

```diff
@@ -68,39 +81,220 @@ public DsStorageClient(String serviceURI) {
+     * @throws ApiException if fails to make API call
+     */
+    @Override
+    public void mappingPost(MappingDto mapping) throws ApiException {               
+        try {
+            URI uri = new URIBuilder(serviceURI + "mapping") //Set the full path, then add parameters.                                                               
+               .build();
+            Service2ServiceRequest.httpCallWithOAuthToken(uri,"POST", null, mapping);              
+        }
+        catch(Exception e) {
+            throw new ApiException(e);
+        }                               
+    }
+
+    /**
+     * Get mapping for a specific referenceId
+     * Get entry from the mapping table for the referenceId. If the entry is not found null will be returned. It is not guarantees the entry if it exists, will have the kalturaId set yet.
+     * @param referenceId  (required)
+     * @return MappingDto
+     * @throws ApiException if fails to make API call
+     */
+    @Override
+    public MappingDto getMapping(String referenceId) throws ApiException {        
+        try {
+            URI uri = new URIBuilder(serviceURI + "mapping") //Set the full path, then add parameters.                                                                
+                    .addParameter("referenceId",referenceId)                       
+                    .build();
+            return Service2ServiceRequest.httpCallWithOAuthToken(uri,"GET", new MappingDto(), null);              
+        }
+        catch(Exception e) {
+            throw new ApiException(e);
+        }                                                
+    }
+        
+    /**
+     * Get a list of minimal records having a referenceId after a given lastModified time
+     * Extract a list of records with a given batch size by origin and mTime larger than input. 
+     * The records will only have the id, mTime, referenceId and kalturaId fields. This means that no actual data can be retrieved through this endpoint. It can however be used  for operations where the data from the record isn&#39;t needed. Such as updating Kaltura IDs for records, which is done with referenceId and kalturaId only. 
+     * @param origin The origin to extract records for (required)
+     * @param maxRecords Number of records to extract. (required)
+     * @param mTime Only extract records after this mTime. (optional, default to 0l)
+     * @return List&lt;DsRecordMinimalDto&gt;
+     * @throws ApiException if fails to make API call
+     */
+    @Override
+    public List<DsRecordMinimalDto> getMinimalRecords (String origin, Integer maxRecords, Long mTime) throws ApiException {   
+        try {
+            URI uri = new URIBuilder(serviceURI + "records/minimal") //Set the full path, then add parameters.                                                               
+                    .addParameter("origin",origin)
+                    .addParameter("maxRecords",""+maxRecords)
```

Comment:
An empty string seems to have entered this line

Reply:
Do not understand....

Reply:
After your comma, you do ""+maxRecords. If thats for casting to a string, then why not use .toString() instead?